images/cli/Dockerfile

```diff
@@ -10,6 +14,8 @@ ENV PHP_EXTENSION_PDO_PGSQL=1
 
 # PHP ini variables
 ENV PHP_INI_XDEBUG__PROFILER_ENABLE_TRIGGER=1
+ENV PHP_INI_XDEBUG__MODE=profil
```

Comment:
Can you make a different PR for theses variables, so we can merge it already.

Reply:
Please see https://github.com/Digital-Peak/DPDocker/pull/22